lib/pagseguro/items.rb

```diff
@@ -24,7 +30,9 @@ def <<(item)
     # Verify if the item is already included to item list.
     # Returns boolean.
     def include?(item)
-      @store.find {|stored_item| stored_item.id == ensure_type(Item, item).id }
+      @store.detect do |stored_item|
+        stored_item.id == item.id && stored_item.description == item.description
```

Comment:
Currently we have compare only id, but the customer order:
> Tem que ser possível adicionar vários items, mesmo com o ID repetido

This task (_Permitir a inclusão de itens com mesmo id_) can be follow through central.